### PR TITLE
Appdev 10356 appdev 10357 new vm

### PR DESCRIPTION
**Pull request by sharonluong**
This PR includes dependency updates for moving Jitterbug to PHP 7.4 and Solr 8.11. The VM setup script had to be updated, as well as the CircleCi configuration. Composer updates also had to be done: mainly mockery was updated but then I also accidentally deleted it and re-added it, hence the difference in spaces. Lastly I updated the readme which was missing some tips.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,14 +72,23 @@ php artisan db:seed --class=UsersTableSeeder
...
 	1. Point your favorite web browser to http://homestead.test:8983/solr
 	2. Use the Solr "Core Selector" menu to select the jitterbug-items core.
 	3. Click the Dataimport button under the Core Selector menu.
 	4. Underneath the Execute button, check the Auto-Refresh Status checkbox.
 	5. Click Execute.
 	6. When jitterbug-items is finished indexing, repeat these steps for each core.
-
+- If the browser cannot connect, open virtualbox and click on your VM. 
+  Then click the settings icon and click on Network, and expand the Advanced dropdown.
+  Click on port forwarding and add 8983 to the list by clicking on the green plus in the top right.
```
Comment (cazzerson) [inline README.md:90]: Is this an issue with Homestead? Is that port set to be forwarded somewhere?

**Review by cazzerson — COMMENTED** [review]
This looks great--just one question in a comment. Thanks!

File: README.md (hunk 1 above)
Comment (sharonluong) [reply README.md:90]: I'm not sure what happened, but I think with the last upgrade or so accessing the solr admin page stopped working. It wouldn't connect, and after troubleshootng with @bbpennel he suggested adding the port manually in virtualbox and it worked. I realized that I didn't have it written down anywhere so I thought i should add it.

Comment (cazzerson) [reply README.md:90]: OK, that sounds fine, thanks.

**Review by cazzerson — APPROVED** [review]

**Review by cazzerson — APPROVED** [review]

**Pull request merged**